clients/ui/frontend/docs/testing.md

```diff
@@ -1,3 +1,27 @@
 # Model Registry UI Testing
 
-[TBD]
\ No newline at end of file
+## Cypress Tests
```

Comment:
Just one nit, can you update the `CONTRIBUTING.MD` file?
Thre're the `Debugging and Testing` section, there you can put something like:

```
See [frontend testing guidelines](docs/testing.md) for more information.
```